### PR TITLE
fix: polish the endpoint list graph

### PR DESCRIPTION
**Pull request by Fine0830**
Polish the endpoint list graph.

Screenshot

Signed-off-by: Qiuxia Fan <qiuxiafan@apache.org>

### CODE REVIEW

File: src/locales/lang/en.ts [hunk 1]
```diff
@@ -152,6 +152,7 @@ const msg = {
   text: "Text",
   query: "Query",
   postgreSQL: "PostgreSQL",
+  endpointTips: "The table shows up to 20 pieces of data.",
```
Comment (wu-sheng) [inline src/locales/lang/en.ts:155]:
```suggestion
  endpointTips: "The table shows up to 20 endpoints.",
```

Could we move this text next to the search bar?

Comment (Fine0830) [reply src/locales/lang/en.ts:155]: Done.

**Comment by wu-sheng** [comment]
Could we change the red to a kind of light red?

**Comment by Fine0830** [comment]
> Could we change the red to a kind of light red?

What do you think of this?

**Comment by wu-sheng** [comment]
Yes, this works.

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**